src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -752,7 +781,10 @@ OSXOutput::Close() noexcept
 {
 	AudioOutputUnitStop(au);
 	AudioUnitUninitialize(au);
-	delete ring_buffer;
+	if (ring_buffer) {
+		delete ring_buffer;
+		ring_buffer = nullptr;
```

Comment:
Assigning `nullptr` here is not necessary. This just hides possible use-after-free bugs.

Reply:
Done in #598